### DISCUSSION TITLE
Priceline override from the series elements

### DISCUSSION THREAD

**Discussion by christose** [tradingview/lightweight-charts#681] [General] [answered]
Hello,

I have an issue with the priceLine visibility. The price line is going behind the series, line or candlesticks, it gets override from them. However, I've checked both charts in tradingview and this is not happening there. See screenshots below.

![image](https://user-images.githubusercontent.com/8969889/105164699-1d151700-5b1e-11eb-980c-53dd65aa363c.png)
![image](https://user-images.githubusercontent.com/8969889/105164710-20a89e00-5b1e-11eb-8825-582a5412e59f.png)

**Comment by timocov** [comment, marked as answer]
It seems that it's [intended right now](https://github.com/tradingview/lightweight-charts/blob/5e6327f6acaa157a4a3889dae7f71920fa284696/src/model/series.ts#L359-L361), but I think it's strange and we have to change that. Can you create an issue for that please?

Meantime, @kirchet don't you remember why custom price lines are behind the series?

**Reply by christose** [reply]
Oh that code you are pointing its the implementation we've done together last summer, with the custom price line with title.

Yes of course, I will create an issue in the morning.

**Reply by christose** [reply]
Issue created!
